websites/A/AnimeVsub/presence.ts

```diff
@@ -0,0 +1,127 @@
+import { ActivityType, Assets, getTimestamps } from 'premid'
+
+const presence = new Presence({
+  clientId: '1348239315767132160',
+})
+
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+
+let strings: { pause: string, play: string } | null = null
+let oldLang: string | null = null
+
+async function getStrings() {
+  const lang = await presence.getSetting<string>('lang').catch(() => 'en')
+  return lang === 'vi'
+    ? { pause: 'Tạm Dừng', play: 'Đang Phát' }
+    : { pause: 'Pause', play: 'Playing' }
```

Comment:
Rather than supporting only two languages, use the built-in `getStrings` function to get the common strings for play/pause.

Reply:
Not resolved. See other recently-updated activities with multi-language support for reference.